common/views/components/ExhibitionPromo/ExhibitionPromo.js

```diff
@@ -55,7 +55,7 @@ const ExhibitionPromo = ({
       <div className={`
           flex flex--column flex-1 flex--h-space-between
           ${spacing({s: 2}, {padding: ['top']})}
-          ${spacing({s: 3}, {padding: ['left', 'right']})}
+          ${spacing({s: 2}, {padding: ['left', 'right']})}
           ${spacing({s: 4}, {padding: ['bottom']})}
         `}>
 
```

Comment:
Do you think we should be getting these into the state of the `BasePage` - where we have a `BasePromo` that takes different components into different slots?

Reply:
I wondered. Maybe.